### PR TITLE
feat: fluent logger and winston upgrade

### PR DESCRIPTION
**Pull request by JamilOmar**

### CODE REVIEW

File: lib/cli/init.ts [hunk 1]
```diff
@@ -34,9 +34,14 @@ export function init(): void {
 
     let logDirectory: string = _.get(config, 'lsc.Log.Path');
     let fluentD = _.get(config, 'lsc.Log.FluentD', {});
+    let format = _.get(config, 'lsc.Log.Format', {});
 
-    global.LabShare.Logger = new Logger({
+    // default logger format - for text
+    global.LabShare.LoggerFormat = LoggerFormat;
```
Comment (KalleV) [inline lib/cli/init.ts]: Can we avoid setting another global? Ideally, the logger instance would be accessed as a singleton.

File: lib/log/logger.ts [hunk 2]
```diff
@@ -27,75 +26,96 @@ function mkdirSync(path: string): void {
...
-                timestamp: true,
-                json: false,
-                maxfiles: 5,
-                maxsize: 10485760,
-                level: 'info'
-            }));
+    // special case: if json is not defined , use by default the custom format 
+    if (options.format.json === false) {
+        // going to global
+        styleFormats.push( global.LabShare.LoggerFormat);
```
Comment (KalleV) [inline lib/log/logger.ts]: Accessing a global here will break the Hedwig Storage api project which doesn’t use LSC as a CLI.

File: lib/cli/init.ts (hunk 1 above)
Comment (JamilOmar) [reply lib/cli/init.ts]: @KalleV  ok I will remote the Format override and define it as a static method

**Pull request merged**